tmpfiles.sh

```diff
@@ -410,21 +588,13 @@ _w() {
 }
 
 _z() {
-	# Set ownership, access mode and relabel security context of a file or
-	# directory if it exists. Lines of this type accept shell-style globs in
-	# place of normal path names.
-	[ "${CREATE}" -gt 0 ] || return 0
-
-	relabel "$@"
+	echo "Ignoring recursively *"
```

Comment:
This makes us non-compliant with the spec: https://www.freedesktop.org/software/systemd/man/tmpfiles.d.html.